mobile/test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -194,64 +142,9 @@ class EnvoyConfigurationTest {
     assertThat(resolvedTemplate).contains("test_config")
 
     // Cert Validation
-    assertThat(resolvedTemplate).contains("custom_validator_config")
+    assertThat(resolvedTemplate).contains("trusted_ca:")
 
     // Proxying
     assertThat(resolvedTemplate).contains("&skip_dns_lookup_for_proxied_requests false")
   }
-
-  @Test
-  fun `configuration resolves with alternate values`() {
```

Comment:
I would think we would still want coverage for the generated YAML with alternate builder values. Is that coverage provided elsewhere?

Reply:
+1

Reply:
sorry was focused on the template vlaues not boolean